.github/workflows/deploy-stg-ecs.yml

```diff
@@ -66,9 +66,3 @@ jobs:
           npm ci
           sls deploy --stage stg
           sls deploy --stage dotcomstg
-
-      - name: Deploy Lambdas
-        run: |
-          npm ci
-          sls deploy --stage stg
-          sls deploy --stage dotcomstg
```

Comment:
Is it okay to remove this when deploying to staging? @GuruPKK

Reply:
updated